ArgoTests/Tests/ExampleTests.swift

```diff
@@ -13,7 +12,7 @@ class ExampleTests: XCTestCase {
 
   func testJSONWithRootObject() {
     let json = JSON.parse <^> JSONFileReader.JSON(fromFile: "root_object")
-    let user: User? = json >>- { $0 <| "user" >>- User.decode }
+    let user: User? = json >>- { ($0 <| "user").value }
```

Comment:
I would like to use the decode function but as it is now, it can't take out root keys ... we might possibly want to make decode functions that also take JSON in instead of AnyObject